app/code/community/Bolt/Boltpay/Block/Checkout/Boltpay.php

```diff
@@ -307,13 +307,23 @@ public function getCartDataJs($checkoutType = 'multi-page')
             // before Bolt defined event JS to give merchants the
             // opportunity to do full overrides
             //////////////////////////////////////////////////////
-            $check = Mage::getStoreConfig('payment/boltpay/check');
-            $onCheckoutStart = Mage::getStoreConfig('payment/boltpay/on_checkout_start');
-            $onShippingDetailsComplete = Mage::getStoreConfig('payment/boltpay/on_shipping_details_complete');
-            $onShippingOptionsComplete = Mage::getStoreConfig('payment/boltpay/on_shipping_options_complete');
-            $onPaymentSubmit = Mage::getStoreConfig('payment/boltpay/on_payment_submit');
-            $success = Mage::getStoreConfig('payment/boltpay/success');
-            $close = Mage::getStoreConfig('payment/boltpay/close');
+            if (($checkoutType === 'admin') && !Mage::getStoreConfig('payment/boltpay/use_javascript_in_admin')) {
```

Comment:
How about we always keep these empty if checkouttype is admin? (so we don't add `use_javascript_in_admin`). 
We can add flag later when really needed, but once we add flag it's our burden to maintain.

Reply:
@daisy1754

Currently, I have set the default value of this to be `false`.

So, for all active merchants, they will have to actively turn this on. If they actively turn this on, it would probably be for a reason good reason, like calling an external API.

I do not see this as causing a superfluous burden, but rather staying consistent with our docs and adding a simple point of override if when necessary for those who actively enable this feature.

What do you think?